src/en/mobile/android/development/general-navigation.md

```diff
@@ -0,0 +1,179 @@
+    android:fontFamily="sans-serif" 
+    android:textSize="14sp"
+    android:text="Exemple"
+&#47;&gt;</code></pre>
+
+
+**Unvalid example&nbsp;:** 
+<pre><code class="xml">&lt;TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:fontFamily="sans-serif" 
+    android:textStyle="italic" 
+    android:textSize="14sp"
+    android:text="Exemple"
+&#47;&gt;</code></pre>
+
+
+<pre><code class="xml">&lt;TextView
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:fontFamily="sans-serif" 
+    android:textSize="10sp"
+    android:text="Exemple"
+&#47;&gt;</code></pre>
+
+**Reference <abbr>WCAG</abbr>&nbsp;:**  
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#text-spacing">1.4.12 Text Spacing</a>
+
+
+<br/><br/>
+
+## Make UI with timeout adaptable 
+
+**Target&nbsp;:** everyone and in particular people with visual and/or motor impairments.
+**When&nbsp;:** from the design phase and during development.
+
+**Description&nbsp;:**
+
+In some applications, the UI may change after a timeout. For example, with video players:  control buttons associated with the video disappear after about ten seconds without interaction. 
+
+This automatic UI update is an issue for accessibility, cause a user could need more time to interact with the elements. This timeout must be adaptable according to the needs of each user. 
+
+An accessibility option named "Time to take action" exists since Android 10, and allows to define its own timeout. By calling this option from the application, it is possible to adapt its interfaces with timeout to the user.
+
+It is possible to exploit this option through the getRecommendedTimeoutMillis() function. This method supports user-defined timeouts for interactive and non-interactive user interface elements. The return value is influenced by both user preferences (Time to take action) and accessibility service APIs.
+
+**To be verified&nbsp;:**
+
+- A UI with a timeout has a configurable delay.
+
+**Reference <abbr>WCAG</abbr>&nbsp;:**  
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#timing-adjustable">2.2.1 Timing Adjustable</a>
+
+
+
+## Accessibility events & custom views
+
+**Target&nbsp;:** everyone and in particular people with visual and/or motor impairments.
+**When&nbsp;:** during development.
+
+**Description&nbsp;:**
+There are a lot of events related to the Accessibility API in Android. You can manipulate these events in the same way as other events. It is important to know that these events exist, allow to overload the vocalization of a component, and to enrich custom components that are not sufficiently accessible.
+
+For more information, look at the links below.
+
+**Links&nbsp;:**
+
+- [`AccessibilityEvents`](http://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html)
+- [Construire une vue custom accessible](https://developer.android.com/guide/topics/ui/accessibility/custom-views)
```

Comment:
This line and the next would need to either be translated or to state that they're in French and/or point to French-speaking resources.